combat-trainer.lic

```diff
@@ -1097,7 +1099,13 @@ class TrainerProcess
     when 'Meraud'
       meraud_commune(game_state)
     when 'Recall'
-      bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?
+      bput("recall #{game_state.npcs.first}", 'Roundtime', 'You are far too occupied') unless game_state.npcs.empty?   
+    when 'Research Augmentation'
+      bput('meditate research monkey', 'You clear ') if DRStats.barbarian?
```

Comment:
I guess Barb Research X would be OK, and the reason I chose to check except for the others is because probably how I chose to name them. The others are kind of given, this is "Research Warding", etc, where people could think "I need to Research Warding because I use magic". I mean, it does make sense that people should be able to figure it out on their own to a certain point, but I was trying to keep the name of the ability shorter to match the others, and I am not really sure what the issue is with checking if the person is a Barbarian before putting it to the screen. Though I guess what could be the case is people would have bloated YAMLs because they have no real idea of what they're doing.